integration_tests/pages/page.ts

```diff
@@ -203,8 +191,6 @@ export default abstract class Page {
         const { actual, expected } = Helpers.parseHtml(fieldErrorElement, `Error: ${error.message}`)
         expect(actual).to.equal(expected)
       })
-
-      cy.get(`[name="${error.field}"]`).should('have.class', 'govuk-input--error')
```

Comment:
For a checkbox, the error class is on a form group containing the input.

Reply:
Just seen this. Can you mention this in the commit message since there's a change in method functionality (not just use) here?

I suppose the bit above is testing that the field error is present, so this should be okay, unless we can come up with a generic way to test that the error class is present